sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -59,11 +59,14 @@ static void _stale_nc(kernel_pid_t iface, ipv6_addr_t *ipaddr, uint8_t *l2addr,
             gnrc_ipv6_netif_t *ipv6_iface = gnrc_ipv6_netif_get(iface);
             if ((ipv6_iface == NULL) || (ipv6_iface->flags & GNRC_IPV6_NETIF_FLAGS_ROUTER)) {
```

Comment:
May be unrelated to this PR, but shouldn't this line actually look like the following:
```c
if ((ipv6_iface != NULL) &&
    (ipv6_iface->flags & (GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN | GNRC_IPV6_NETIF_FLAGS_ROUTER))) {
```

Otherwise, I do not understand why you would want to add an `nc_entry` to an interface, that is not IPv6.
@authmillenon  ?